tests/test_image.py

```diff
@@ -53,20 +54,40 @@ def testImageSqrt(self):
         self.assertTrue(np.allclose(self.ims.sqrt().asnumpy(), np.sqrt(self.ims_np)))
 
     def testImageTranspose(self):
+        # test method and abbreviation
         self.assertTrue(
             np.allclose(
-                self.im.flip_axes().asnumpy(), np.transpose(self.im_np, (0, 2, 1))
+                self.im.T.asnumpy(),
+                self.im.transpose().asnumpy(),
+                np.transpose(self.im_np, (0, 2, 1)),
             )
         )
 
-        # This is equivalent to checking np.tranpose(..., (0, 2, 1))
+        # Check individual imgs in a stack
         for i in range(self.ims_np.shape[0]):
+            self.assertTrue(
+                np.allclose(self.ims.T[i], self.ims.transpose()[i], self.ims_np[i].T)
+            )
 
-            self.assertTrue(np.allclose(self.ims.flip_axes()[i], self.ims_np[i].T))
-
-            # Check against the contruction.
+    def testImageFlip(self):
+        axes = list(combinations([1, 2], 2))
```

Comment:
```
>>>from itertools import combinations
>>>list(combinations([1, 2], 2))
[(1, 2)]
```
This and some of the lines below don't seem right.

Reply:
I think we'll want to cover all combinations, plus the `0` cases.  I think your `0` tests look fine.

You can just write them out as `[1,2, (1,2), (2,1)]`, or you can use the `powerset` for combinations I added to our utils.

Reply:
I think `powerset` is good, especially since `(1,2)` and `(2,1)` do the same thing in `flip`. Thanks. That's also what's used in the Volume flip test